### ISSUE TITLE
Priority View does not limit the width of text 

### ISSUE THREAD

**Issue by TechSupport26** [lightningrodlabs/acorn#204]
UI issue with limiting the width of text in Priority View. v2.1.1-alpha

![image](https://user-images.githubusercontent.com/65631811/199421228-8fb193ae-53e7-4a59-807e-570003967a94.png)

**Comment by TechSupport26** [comment]
I would also like to add that if the Map View is not working (for me I made a card with a lot of characters in it) you can go to Priority View and open the card and remove most of the text.

**Comment by pegahvaezi** [comment]
Yes there are currently no UX implementations for character limit on Outcome Statement. This is in the roadmap for acorn, but currently not one of the high priority items.

**Comment by TechSupport26** [comment]
I just meant for the text to wrap around to the next line when it reached the edge of the screen.
There is no rush.
